docs/docs_translations/plugins.pot

```diff
@@ -117,7 +117,7 @@ msgid "Open *Twilio* Plugin and activate it."
 msgstr ""
 
 #: ../plugins/twilio.rst:23
-msgid "Copy the *Account Sender ID* from Twilio account and paste it to *Account Sender ID* field in the Twilio Plugin configuration."
+msgid "Copy the *Messaging Sender ID* from Twilio account and paste it to *Messaging Sender ID* field in the Twilio Plugin configuration."
```

Comment:
@escopecz Account sender ID is different thing we had added Messaging Service SID and it will be used in place of phone number in feature tab.
So Account Sender ID will be remain as it is and phone number will be replace by Messaging Service SID.

Reply:
@dadarya0 can you rephrase the message for me?

Reply:
Or even create a new PR with the changes needed? I don't get the change clearly.

Reply:
In feature setting of Twilio configuration, **Sending Phone Number** is replace by **Messaging Service SID**.

![image](https://github.com/mautic/user-documentation/assets/48244990/7377744e-0084-4267-bd17-6d796e38d21c)

**How to generate Messaging Service SID :-**

1. Login to twilio account
2. In search box, serach for Messaging services  and select Messaging services from search result

![image](https://github.com/mautic/user-documentation/assets/48244990/275bf9a8-41d7-4977-9f6a-7bf209fb70a9)


3. Click on Create Messaging Service button

4. Input all required field and configure it

5. Now user will get Messaging Service SID, now copy and use it in acs.

Reply:
This whole process, including the explanation of 'input all required fields and configure it' step by step, is needed for this PR to be mergeable.